app/javascript/pages/getAssistance/additional-resources.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react"
+import Layout from "../../layouts/Layout"
+import withAppSetup from "../../layouts/withAppSetup"
+
+const AdditionalResources = () => {
+  return <Layout title={"Additional Resources"}>{<h1>Additional Resources</h1>}</Layout>
```

Comment:
Can we add localization for the titles and headings? This will give us more confidence that the language pages are loading properly. We can also create the scaffolds for the unit tests by looking for these headings and keep our coverage up.